java/java.source.ant/src/org/netbeans/modules/java/source/ant/ProjectRunnerImpl.java

```diff
@@ -122,8 +122,7 @@
     private static final RequestProcessor RP = new RequestProcessor(ProjectRunnerImpl.class);
     
     public boolean isSupported(String command, Map<String, ?> properties) {
-        return BuildArtifactMapper.isCompileOnSaveSupported() &&
-               locateScript(command) != null;
+        return locateScript(command) != null;
```

Comment:
Without this change, @jlahoda, the Ant runner refused to perform `run` and `debug` actions. I am sure the check is here for a reason, but it is probably not correct to prevent `run`/`debug` just because we don't have `nbjavac` installed.